### PR TITLE
Improve Obsidian Publish docs

### PR DESCRIPTION
**Pull request by marcusolsson**
This PR aims to improve the docs for Obsidian Publish. This PR depends on, and should be merged after, #200.

I've left the docs on setting up custom domains pretty much as is for now. I'd need to configure this myself to verify the steps necessary, before doing any major updates to it.

### CODE REVIEW

File: en/Obsidian Publish/Introduction to Obsidian Publish.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+alias: [Obsidian Publish]
+---
+
+Obsidian Publish is a cloud-based service where you can publish your notes to the world without leaving your favorite text editor.
```
Comment (ericaxu) [inline en/Obsidian Publish/Introduction to Obsidian Publish.md:5]: Does "your favorite text editor" mean Obsidian? I get the slight humor but some people might get the wrong idea if their favorite is VS Code or something 🤔

File: en/Obsidian Publish/Introduction to Obsidian Publish.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Select the notes you want to share with the world, press 'Publish', and we'll post it to publish.obsidian.md/your-site.
```
Comment (ericaxu) [inline en/Obsidian Publish/Introduction to Obsidian Publish.md:7]: wondering whether `'Publish'` should be `"Publish"` (English) or `**Publish**` (UI instruction convention as in the Sync PR). What do you think?

File: en/Obsidian Publish/Introduction to Obsidian Publish.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+If you're reading this on [help.obsidian.md](https://help.obsidian.md), you're already looking at a site that uses Obsidian Publish. You can see the vault on [GitHub](https://github.com/obsidianmd/obsidian-docs).
+
+## Learn more
+
+- If you a new Obsidian Publish user, learn how to [[Set up Obsidian Publish]].
+- If you have more than one site, learn how to [[Manage sites]].
```
Comment (ericaxu) [inline en/Obsidian Publish/Introduction to Obsidian Publish.md:14]: maybe add a scenario for people who want to customize how they site looks & work?

File: en/Obsidian Sync/Limitations.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+This page lists some of the common questions related to quotas and limitations for [[Introduction to Obsidian Sync|Obsidian Sync]].
```
Comment (ericaxu) [inline en/Obsidian Sync/Limitations.md:1]: Why is this file here? Thought I merged it in #200

File: en/Obsidian Publish/Security and privacy.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+You choose which notes you want to publish to [[Introduction to Obsidian Publish|Obsidian Publish]]. The rest of your notes stay safe in your vault.
```
Comment (ericaxu) [inline en/Obsidian Publish/Security and privacy.md:1]: maybe touch one that one can set one or more passwords to protect their entire site (but not certain files or folders)

File: en/Obsidian Publish/Introduction to Obsidian Publish.md (hunk 1 above)
Comment (marcusolsson) [reply en/Obsidian Publish/Introduction to Obsidian Publish.md]: Good point! I've removed it for now.

File: en/Obsidian Publish/Introduction to Obsidian Publish.md (hunk 2 above)
Comment (marcusolsson) [reply en/Obsidian Publish/Introduction to Obsidian Publish.md]: Good catch! I wanted to describe the workflow at a high-level, so I'm not referring to a specific UI element here, but rather the action of publishing something. I switched to double-quotes.

File: en/Obsidian Publish/Introduction to Obsidian Publish.md (hunk 3 above)
Comment (marcusolsson) [reply en/Obsidian Publish/Introduction to Obsidian Publish.md:14]: 💯  I'll get on it!

File: en/Obsidian Publish/Security and privacy.md (hunk 5 above)
Comment (marcusolsson) [reply en/Obsidian Publish/Security and privacy.md:1]: 💯  I'll get on it!

Comment (marcusolsson) [reply en/Obsidian Publish/Security and privacy.md:1]: 💯 I'll get on it!

File: en/Plugins/Core plugins.md [hunk 6]
```diff
@@ -29,5 +29,4 @@ Plugins can be managed through the "Settings" button on the bottom left. Some of
 1. [[Open in default app]]
 1. [[Workspaces]]
 1. [[File recovery]]
-1. [[Publish]]
```
Comment (ericaxu) [inline en/Plugins/Core plugins.md:32]: maybe leave it as `[[Introduction to Obsidian Publish|Publish]]` to be consistent with Sync?

Comment (marcusolsson) [reply en/Plugins/Core plugins.md:32]: Good catch!

File: en/Obsidian Publish/Customize your site.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+- `favicon-180x180.png`
+- `favicon-192x192.png`
+- `favicon-196x196.png`
+
+## Use a community theme
+
+To use one of the community themes for your site:
+
+1. Open your vault in the default file explorer for your OS.
+2. Go to the vault settings folder (default: `.obsidian`).
```
Comment (ericaxu) [inline en/Obsidian Publish/Customize your site.md:37]: note for future: might need to link to "how to open hidden folders" in the future as I see people get tripped up a lot on this

Comment (ericaxu) [reply en/Obsidian Publish/Customize your site.md:37]: nothing actionable is required
